src/homework_1/Raising.java

```diff
@@ -0,0 +1,25 @@
+package homework_1;
+
+public class Raising {
```

Comment:
better add App or Application at the end of the class name, because this class have main method